algebra-kernels/README.md

```diff
@@ -0,0 +1,135 @@
+# algebra-kernels
+
+This library contains ready-to-use GPU kernels for the [ginger-lib](https://github.com/HorizenOfficial/ginger-lib) library.
+
+Currently the next algorithm are implemented:
+
+* Multi-scalar multiplication
+* Fast Fourier transform
+* Reduce for third round of Marlin (polycommit)
```

Comment:
A bit more details would be helpful: Which MSM is implemented (I suppose Pippenger in affine coordinates, using batch inversion), which FFT (radix 2, or mixed?), and what does "reduction of third round Marlin" mean (I expect the computation of the new committer keys,i.e. the "many SM",  in each step the dlog reduction?).